src/forms/webhook-notifications/permissions-change.ts

```diff
@@ -0,0 +1,35 @@
+import { AppBindingLocations } from '../../constant/apps';
+import manifest from '../../manifest.json';
+import { GA$DriveActivity, Schema$File, Schema$User, User, WebhookRequest } from '../../types';
+import { h5, hyperlink, inLineImage } from '../../utils/markdown';
+import { postBotChannel } from '../../utils/post-in-channel';
+import { configureI18n } from '../../utils/translations';
+
+import { getMattermostUserFromGoogleEmail } from './get-mm-username';
+
+export async function permissionsChanged(call: WebhookRequest, file: Schema$File, activity: GA$DriveActivity): Promise<void> {
+    const i18nObj = configureI18n(call.context);
+
+    const author = file.sharingUser as Schema$User;
+    const actorEmail = <string>author.emailAddress;
+    let userDisplay = `${author?.displayName} (${actorEmail})`;
+    const mmUser = await getMattermostUserFromGoogleEmail(call, actorEmail) as User;
+
+    if (Boolean(mmUser)) {
+        userDisplay = `@${mmUser.username}`;
+    }
```

Comment:
Done, this was resolved in commit [4884970](https://github.com/mattermost/mattermost-app-google-drive/pull/2/commits/48849707058fdb0000bb78d877cb3c630a459de6)

Reply:
That commit adds a question mark at `mmUser?.message`, which we don't need since typescript already makes sure that's defined. Not required to remove the question mark in this PR, just pointing it out so other new code can be made more readable in this way.